CMSgov/mint-app
# [Easi 4148]op needs, op solutions, op solutions subtask audit meta data

# EASI-4148


## Changes and Description

- Add meta data for these types
   - Operational Need
   - Operational Solution
   - Operational Solution Subtask
   
- Update postman collection

## How to test this change
1. Make changes to these types
2. Assert the meta data is provided as expected for the new types 
   a. You can use the updated postman collection to view the result.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
